recipe/meta.yaml

```diff
@@ -1,12 +1,11 @@
-{% set name = "jupyter-server-proxy" %}
-{% set version = "3.1.0" %}
+{% set version = "3.2.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: jupyter-server-proxy
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/j/jupyter-server-proxy/jupyter-server-proxy-{{ version }}.tar.gz
```

Comment:
I think I'd be opined towards not hardcoding this like that, mainly because the `/j/` part in the url wasn't something I would have guessed should have been updated if jupyter-server-proxy would be renamed or similar perhaps.

I think this is fine though, I know too little about conda-forge practices and such to say what aligns itself well to make maintenance matters easy.

Reply:
There's generally been a trend towards fewer jinja variables... In the case of a hard fork/rename, a new feedstock might well make the most sense, as have _even more_ naming dissonance doesn't help much.

Reply:
Yeah this kind of thing was more prevalent pre-[Grayskull]( https://github.com/conda-incubator/grayskull )

Reply:
:tada: nice thanks for the clarification!